starter-code/index.html

```diff
@@ -54,15 +54,16 @@
             </div>
           </header>
           <section class="article-body"></section>
-          <a href="#" class="read-on">Read on &rarr;</a>
+          <a href="#" class="read-on" id="readOn">Read on &rarr;</a>
+          <a href="#" class="read-on" id="showLess">Show less &rarr;</a>
```

Comment:
Good approach for toggling the text! You could also change the text of a single anchor tag with the jQuery text method.